presto-hive/src/main/java/com/facebook/presto/hive/BackgroundHiveSplitLoader.java

```diff
@@ -645,4 +661,25 @@ private static String getPartitionLocation(Table table, Optional<Partition> part
         }
         return partition.get().getStorage().getLocation();
     }
+
+    private static Optional<Domain> getPathDomain(TupleDomain<HiveColumnHandle> effectivePredicate)
+    {
+        if (!effectivePredicate.getColumnDomains().isPresent()) {
```

Comment:
What are you trying to achieve here? I looked at getColumnDomains, and it has a comment that says `Available for Jackson serialization only!` - doesn't look like it's meant for other uses. Also, there are almost no other usages of this function.